docs/guide/writing/xml.rst

```diff
@@ -0,0 +1,63 @@
+===================
+Introduction to XML
+===================
+
+What is XML?
+============
+
+Similar to reStructuredText, Markdown, and HTML, XML is a markup language. Unlike other markup languages, it is not predefined and you must define your own tags and attributes. It's different from other markup languages in that instead of being used to present content on a page, it distributes that page's data over the Internet.
```

Comment:
Not 100% sure I personally agree with some of these definitions, but I think we're ok to merge and iterate on this.

@writethedocs/core any objections?

Reply:
I would rewrite this section completely. First of all, it is not clear what XML actually is and why should I use it. Second, the section is too wordy (markup language everywhere) and there are styling inconsistencies (e.g. it is and it's).

Reply:
> 
> 
> I would rewrite this section completely. First of all, it is not clear what XML actually is and why should I use it. Second, the section is too wordy (markup language everywhere) and there are styling inconsistencies (e.g. it is and it's).

I can make a quick edit on this section :)

Reply:
> I can make a quick edit on this section :)

That would be great! I will post my suggestions shortly